spec/models/user_spec.rb

```diff
@@ -0,0 +1,62 @@
+require 'rails_helper'
+
+RSpec.describe User do
+  subject(:user) { build(:user) }
+  let(:plain_user) { create(:user) }
+
+  it_behaves_like 'a Hyrax User'
+
+  it "has a display name" do
+    expect(user.display_name).not_to be_empty
+  end
+
+  it '#to_s yeilds the #display_name' do
+    expect(user.to_s).to eq user.display_name
+  end
+
+  describe 'roles' do
+    it 'is emplty for a new user' do
+      new_user = create(:user)
+      expect(new_user.roles).to be_empty
```

Comment:
Could we do `expect(plain_user.roles).to be_empty` instead? This has the advantage of ensuring that `plain_user` doesn't accidentally become non-plain in the future, invalidating other tests.

Reply:
👍